fixtures/attribute.rb.sem.uast

```diff
@@ -0,0 +1,53 @@
+{ '@type': "module",
+   '@role': [Identifier, Module, Statement],
+   file: { '@type': "send_assign",
+      '@token': "foo",
```

Comment:
Should be a `uast:Identifier`

Reply:
Fixed in next commit (applies the same I said above).